SwagLyricsBot/dev_commands.py

```diff
@@ -1,6 +1,7 @@
 import os
 import time
 from datetime import timedelta
+from random import getrandbits as rb
```

Comment:
Just use `random.randint`. In case there's a third person you can just `randint(0, 2)`.

Reply:
getrandbits is faster https://stackoverflow.com/a/6824868/9044659

Reply:
Negligible improvement. This function isn't going to be called a lot and overall, doesn't add much performance. 500 ns vs 50 ns won't matter anyway, this is not a loop.

Reply:
fair but there's negligible advantage to using randint if at all, so I don't think it matters. Plus if there is a third member, we'd have to edit the other parts too 🤷‍♂️